cli/azd/cmd/pipeline.go

```diff
@@ -114,6 +116,8 @@ func newPipelineConfigAction(
 	credentialProvider account.SubscriptionCredentialProvider,
 	azdCtx *azdcontext.AzdContext,
 	env *environment.Environment,
+	accountManager account.Manager,
+	_ auth.LoggedInGuard,
```

Comment:
This pattern needs to be revisited. My current thinking is to combine auth and account manager into a single component, and the prompt for account subscriptions should obviously check the auth first. This refactoring is a larger change, so I'm going to create a follow-up issue separately.

Reply:
#2062